README.md

```diff
@@ -105,7 +105,7 @@ install the command line interface and swap client service.
 ```
 git clone https://github.com/lightninglabs/loop.git
 cd loop/cmd
-go install ./...
+go install ./... # Note this is a `...` wildcard
```

Comment:
>was accidentally trying go install ./.. and it seemed perfectly happy aside from not outputting any artifacts in the expected place

Does this comment make that part more clear?

Reply:
would have saved me a few minutes yeah, up to you if it's worth it. Feel free to close, just a personal anecdote.

Reply:
What i try to understand is how that comment that you add in this pr would've saved you the minutes? I would expect the comment to either describe the install location or describe the verbose flags.